com.unity.netcode.gameobjects/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -366,21 +392,56 @@ protected NetworkBehaviour GetNetworkBehaviour(ushort behaviourId)
 
         internal void InternalOnNetworkSpawn()
```

Comment:
I like the fact that now we are relying on `OnNetworkSpawn` and `OnNetworkDespawn` now <3